src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.DecryptToken.cs

```diff
@@ -23,7 +23,7 @@ public partial class JsonWebTokenHandler : TokenHandler
         /// <param name="configuration">The <see cref="BaseConfiguration"/> to be used for validating the token.</param>
         /// <param name="callContext"></param>
         /// <returns>The decoded / cleartext contents of the JWE.</returns>
```

Comment:
Consider updating the comments in the return fields here and below to indicate a valid result on success or an error on failure.